sunpy/map/sources/stereo.py

```diff
@@ -71,17 +76,20 @@ def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an COR image"""
         return header.get('detector', '').startswith('COR')
 
+    def _get_mpl_normalizer(self):
+        """Returns a Normalize object to be used with XRT data"""
+        return PowerNorm(0.5, self.data.min(), self.data.max())
+
 class HIMap(GenericMap):
     """HI Image Map definition"""
 
     def __init__(self, data, header, **kwargs):
 
         GenericMap.__init__(self, data, header, **kwargs)
-
-        self._name = self.observatory + " " + self.detector + " " + str(self.measurement)
+        self.meta['wavelnth'] = np.nan
```

Comment:
because we currently put 0.0 there which does not make sense. The measurement is "white-light" which implies broadband so wavelength does not make sense. A nan suggests that we did not forget to fill it in at least. I'm not super happy with this but it is better than before.